### PR TITLE
983 update highlights to include distributed training

### LINKED ISSUE #983: Add distributed training to highlights doc and charts
**Is your feature request related to a problem? Please describe.**
As we already completed some distributed training experiments on NVIDIA NGC nodes, need to update highlights doc to include this part according to our plan.

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #983 .

### Description
This PR updated highlight features to include distributed training and charts.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh --codeformat --coverage`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Review by ericspod — APPROVED** [review]

**Comment by wyli** [comment]
in the 3rd plot of the last figure, y axis range could be 0.5 ~1.0 instead of 0.0 ~ 1.0 to improve the readability

**Comment by Nic-Ma** [comment]
> in the 3rd plot of the last figure, y axis range could be 0.5 ~1.0 instead of 0.0 ~ 1.0 to improve the readability

Hi @wyli ,

Thanks for your review here.
It starts from 0 to show the `distributed caching` time, with more processes, caching is faster based on partition.

**Comment by wyli** [comment]
that is already obvious in the 2nd plot. I think the last figure should highlight the model quality change against time. and model quality under 0.5 is not very interesting, so we can `ax.set(ylim=(0.5, 1.0))` to display the interesting part

**Comment by Nic-Ma** [comment]
Hi @wyli ,

The second chart doesn't show the cache time, it's time for every epoch, we do caching before the first epoch.
The first epoch takes more time due to PyTorch pre-fetching, not caching.

Thanks.

**Comment by wyli** [comment]
> Hi @wyli ,
> 
> The second chart doesn't show the cache time, it's time for every epoch, we do caching before the first epoch.
> The first epoch takes more time due to PyTorch pre-fetching, not caching.
> 
> Thanks.

that is quite misleading, why not show the caching time in the 2nd?

**Comment by Nic-Ma** [comment]
> > Hi @wyli ,
> > The second chart doesn't show the cache time, it's time for every epoch, we do caching before the first epoch.
> > The first epoch takes more time due to PyTorch pre-fetching, not caching.
> > Thanks.
> 
> that is quite misleading, why not show the caching time in the 2nd?

Caching time is not in the first epoch, and actually we don't include caching time in all MONAI charts based on epochs.
Seems distributed caching time is misleading, let's don't show it, I can try to change the Y range to [0.5, 1.0] then.
Sounds good?

Thanks.

**Comment by wyli** [comment]
> > > Hi @wyli ,
> > > The second chart doesn't show the cache time, it's time for every epoch, we do caching before the first epoch.
> > > The first epoch takes more time due to PyTorch pre-fetching, not caching.
> > > Thanks.
> > 
> > 
> > that is quite misleading, why not show the caching time in the 2nd?
> 
> Caching time is not in the first epoch, and actually we don't include caching time in all MONAI charts based on epochs.
> Seems distributed caching time is misleading, let's don't show it, I can try to change the Y range to [0.5, 1.0] then.
> Sounds good?
> 
> Thanks.

sure, I hope the first plot includes all the caching time. (why do think the caching time is misleading? I feel the caching is also distributed so it's a nice feature for monai, we can sync offline)

**Comment by Nic-Ma** [comment]
Hi @wyli ,

The first chart contains all the time, the second chart is for every epoch time.
Will update soon.
Thanks.

**Comment by Nic-Ma** [comment]
Updated data range to [0.5, 1.0].
Thanks.
